ciri/consensus/tip_selection/exit_probability_randomizer/walker.c

```diff
@@ -67,6 +67,7 @@ static retcode_t find_tail_if_valid(tangle_t *const tangle, exit_prob_transactio
     return ret;
   }
   if (*has_valid_tail) {
+    *has_valid_tail = false;
```

Comment:
But it's already being reset to `false` on `iota_consensus_exit_prob_transaction_validator_is_valid`

Reply:
true